loadtest/control/simplecontroller/controller.go

```diff
@@ -146,30 +157,47 @@ func (c *SimpleController) Run() {
 		// 	waitAfter: 1000,
 		// },
 		{
-			run:       c.viewChannel,
-			waitAfter: 1000,
+			run:          c.viewChannel,
+			waitAfter:    1000,
+			runFrequency: 1,
+		},
+		{
+			run:          c.scrollChannel,
+			waitAfter:    1000,
+			runFrequency: 1,
 		},
 		{
-			run:       c.scrollChannel,
-			waitAfter: 1000,
+			run:          c.leaveChannel,
+			waitAfter:    1000,
+			runFrequency: 1,
 		},
 		{
-			run:       c.leaveChannel,
-			waitAfter: 1000,
+			run:          c.logout,
+			waitAfter:    1000,
+			runFrequency: 20,
 		},
 		{
-			run:       c.logout,
-			waitAfter: 1000,
+			run:          c.login,
+			waitAfter:    1000,
+			runFrequency: 20,
 		},
 	}
 
 	c.status <- control.UserStatus{ControllerId: c.id, User: c.user, Info: "user started", Code: control.USER_STATUS_STARTED}
 
 	defer c.sendStopStatus()
 
+	c.status <- c.signUp()
+	c.status <- c.login()
+
+	cycleCount := 0 // keeps a track of how many times the entire cycle of actions have been completed.
```

Comment:
This is the last detail I was referring to. I suggest we set `cycleCount` to be starting at 1 to avoid running all the actions on the first iteration.